OpenRA.Game/GameRules/Ruleset.cs

```diff
@@ -29,6 +29,7 @@ public class Ruleset
 		public readonly IReadOnlyDictionary<string, MusicInfo> Music;
 		public readonly TileSet TileSet;
 		public readonly SequenceProvider Sequences;
+		public readonly IReadOnlyDictionary<string, MiniYamlNode> ModelSequences;
```

Comment:
We don't have a `*Info` for this: it is up to the mod-defined parser to parse what it wants from the yaml tree.